packages/wrangler/src/pages/dev.ts

```diff
@@ -557,6 +567,32 @@ export const Handler = async ({
 				.map((binding) => binding.toString().split("="))
 				.map(([key, ...values]) => [key, values.join("=")])
 		),
+		services: services
+			.map((serviceBinding) => {
+				const { binding, service, environment } =
+					SERVICE_BINDING_REGEXP.exec(serviceBinding.toString())?.groups || {};
+
+				if (!binding || !service) {
+					logger.warn(
+						"Could not parse Service binding:",
+						serviceBinding.toString()
+					);
+					return;
+				}
+
+				// Envs get appended to the end of the name
+				let serviceName = service;
+				if (environment) {
+					serviceName = `${service}-${environment}`;
```

Comment:
Might be worth logging a "environments are weird and not officially supported" message here until we nail this down.

Reply:
I've added this warning:
![Screenshot 2023-10-11 at 22 35 53](https://github.com/cloudflare/workers-sdk/assets/61631103/6659c2f0-e55a-4e87-a930-9765ed342317)

let me know if you think it's sufficient/appropriate 🙂